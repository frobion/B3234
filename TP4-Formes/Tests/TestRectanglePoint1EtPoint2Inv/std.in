R Rectangle1 6 0 1 3
R Rectangle2 1 3 6 0
LIST
EXIT

