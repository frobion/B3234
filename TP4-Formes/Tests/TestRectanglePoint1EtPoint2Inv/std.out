ERR
# 
OK
R Rectangle2 1 3 6 0

